src/ralph/accounts/models.py

```diff
@@ -188,7 +188,7 @@ def permissions_hash(self):
         Hash for caching is calculated from user ID and its permissions.
         """
         perms_set = frozenset(self.get_all_permissions())
-        key = ':'.join((self.id, hash(perms_set)))
+        key = ':'.join((str(self.id), str(hash(perms_set))))
```

Comment:
Ha! it was one from django's gotchas - if function/property raise ``KeyError`` then django mute this exception by default and returns nothing to template's context therefore our cache key for doesn't work properly - all of users saw same menu